### DISCUSSION TITLE
Migration from 1.3.2 to 2.2.3

### DISCUSSION THREAD

**Discussion by Rutenioh** [MorphiaOrg/morphia#1783] [General]
Hello Justin,

we want to upgrade our mongodb version and we need to upgrade Morphia and we have doubts with the Converter classes we have implemented. We guess we have to migrate to Codec classes but... do you have an example of how to do it?

Any kind of advices are welcome, Justin!

Thanks

**Comment by evanchooly** [comment]
I never got a notification on this post.  I apologize.  I just filed an issue regarding [custom CodecProviders](#1842) that should make 2.3 an easier migration target for you.  I just wrapped up some 2.2.4 issues and am back on 2.3.  The remaining issues are largely mechanical implementations of various operators and so should go quickly.

**Reply by Rutenioh** [reply]
Hi @evanchooly, how is it going the 2.3 version?

**Reply by evanchooly** [reply]
it is *almost* done.  I think there are 4 issues left and only 1 will take a *lot* of work.  I'm hoping to get it done soonish because I have a critter release waiting for this one, too.
